dnf5/context.cpp

```diff
@@ -871,10 +872,24 @@ void RpmTransCB::script_stop(
     [[maybe_unused]] const libdnf5::rpm::TransactionItem * item,
     libdnf5::rpm::Nevra nevra,
     libdnf5::rpm::TransactionCallbacks::ScriptType type,
-    [[maybe_unused]] uint64_t return_code) {
-    active_progress_bar->add_message(
-        libdnf5::cli::progressbar::MessageType::INFO,
-        fmt::format("Stop {} scriptlet: {}", script_type_to_string(type), to_full_nevra_string(nevra)));
+    uint64_t return_code) {
+    switch (return_code) {
+        case RPMRC_OK:
+            // remove the script_start message in case the script finished successfully
+            active_progress_bar->pop_message();
```

Comment:
As @ppisar mentioned, I've also noticed the flickering. There is always a flash of an empty line after the messages is popped and before the next one is printed. I've tried to modify the implementation locally to always just switch on the flag that we have to rewrite the last message next time when adding a new one. It seems the flickering is gone then, but still some edge cases seem need to be covered, like when the message is the last one in the given bar. Let's defer it to a separate issue.